### ISSUE TITLE
Support for RPM distributions?

### ISSUE THREAD

**Issue by braewoods** [Eloston/ungoogled-chromium#1082]
There's a number of RPM based distributions that otherwise have little in common. Examples:

Mageia (Mandrake/Mandriva family)
Fedora/CentOS (RedHat family)
OpenSuse

Is anyone interested in developing packaging scripts for these targets and possibly others that use RPM? OBS happens to support many of these as well so I was thinking it may be worthwhile to expand to these if there's any interested parties.

**Comment by max-privatevoid** [comment]
RPM Fusion has chromium-browser-privacy, which pulls its patches from this github repo
[https://admin.rpmfusion.org/pkgdb/package/free/chromium-browser-privacy/](url)
[https://pkgs.rpmfusion.org/cgit/free/chromium-browser-privacy.git/](url)

**Comment by wchen342** [comment]
I think it was solved in #838?

**Comment by Eloston** [comment]
If chromium-browser-privacy is just ungoogled-chromium but with a different name, then I'm willing to close this.

**Comment by wchen342** [comment]
@qvint Any comments?

**Comment by wchen342** [comment]
Since `chromium-browser-privacy` hasn't been updated for three months and is still on v85 I [created my own version](https://github.com/wchen342/ungoogled-chromium-fedora) based on Fedora's `chromium`. Not sure whether I should make it official though because for now I don't have intention to put it on RPM Fusion.

**Comment by git-Hmmm** [comment]
@Eloston :pray: Please do something regarding this issue.

**Comment by SoftwareRat** [comment]
> Since `chromium-browser-privacy` hasn't been updated for three months and is still on v85 I [created my own version](https://github.com/wchen342/ungoogled-chromium-fedora) based on Fedora's `chromium`. Not sure whether I should make it official though because for now I don't have intention to put it on RPM Fusion.

Archived, so Fedora support is dead

**Comment by networkException** [comment]
there is just nobody maintaining it at the moment, wchen342 stepped down from that role

**Comment by git-Hmmm** [comment]
But Fedora is very popular they should maintain it, I am currently using flatpak but I hope Eloston will bring back support for fedora.

**Comment by networkException** [comment]
Who is they? If there is nobody that is using Fedora and or can maintain a package there's nothing that can be done

**Comment by jstkdng** [comment]
sadly yeah, I believe only those with skin in the game should maintain ungoogled-chromium for their respective distributions, in this case fedora. If you use fedora, why don't you try to maintain a package for it? Nobody is born with the knowledge to main a binary package. Just read the docs and give it a go.

**Comment by git-Hmmm** [comment]
@jstkdng Yeah I will try to but the problem is I don't have enough resources I don't have 100GB free space and I don't have 16GB Ram, so that's the barrier for me and if I would have then also it would be hard for me to build it.

**Comment by PF4Public** [comment]
I'm building in a kvm-guest with 20+GiB, but never seen RAM usage above 8GiB while building Chromium in 9 threads. Chromium + Electron — probbaly :)

**Comment by jstkdng** [comment]
@git-Hmmm you can use the openSUSE Build Service (OBS), it's a public building system and it supports fedora and opensuse build scripts. Check it out. https://en.opensuse.org/openSUSE:Build_Service_Tutorial
